pkgs/ogen-go/ogen/pkg.yaml

```diff
@@ -0,0 +1 @@
+packages: []
```

Comment:
packages is empty.
Seems like `cmdx s` can't generate the code well.
We need to fix the code manually.

https://github.com/ogen-go/ogen/releases

There is no asset in GitHub Releases.

Reply:
Fixed. 7d49c5782be49714c6f1c49983babf868e7a761e